share/spack/templates/container/centos_stream.dockerfile

```diff
@@ -19,10 +19,10 @@ RUN dnf update -y \
         iproute \
         make \
         patch \
-        python38 \
-        python38-pip \
-        python38-setuptools \
+        python3.11 \
+        python3.11-setuptools \
```

Comment:
Is python 3.8 no longer available as well? Or is 3.11 the only one to provide `ensurepip`?

Reply:
The packages are all there, but for some reason I still have to understand sometimes `pip3` is not in PATH. I changed to Python 3.11 since I noticed that:
```
dnf --enablerepo epel groupinstall -y "Development Tools"
```
is also installing that.

Reply:
Looks like `ensurepip` is available since 3.4, so to back up Alec's comment, it seems reasonable to just use `python3 -m ensurepip` there. It is absurd that they have just stopped publishing 3.8 packages.